js/tests/enhancedDate.spec.js

```diff
@@ -0,0 +1,69 @@
+// describing spec (what's being tested)
+describe("enhancedDate", function () {
+
+	// Test each method within object
+
+	// setDate
+	describe("setDate", function () {
+
+		it("Return same date object as date object given.", function () {
```

Comment:
The test title suggests something a little different than what the test itself tests. If you want to test the object returned, you can test that the object give is deep.equal to the object returned. You can also test the result of calling toString on the objects, but it's an additional step beyond testing the method itself.